### DISCUSSION TITLE
Provide sponsorship via GitHub?

### DISCUSSION THREAD

**Discussion by arrigolupori** [PodcastGenerator/PodcastGenerator#663] [General]
I'm interested in PodcastGenerator and would like to provide sponsorship.
However, I don't like to use PayPal, that seems to be the only donate option so far.

Any way that I can support you guys via GitHub itself? :)

**Comment by charabaruk** [comment]
I wasn't even aware that there's anything up to allow PayPal donations. Where did you see that?

I can certainly look into GitHub sponsorship for the project, but it could be a bit of time before that would be all set up.

**Reply by arrigolupori** [reply]
It's at the bottom of the podcast generator website (https://podcastgenerator.net/). I want to support the project because I think it's very valuable and it deserves some level of contribution :)

**Reply by charabaruk** [reply]
Ah. I don't control that website yet and it's at this point fairly out of date. @albertobeta and I have been talking about it in private but so far nothing's happened since I've been constantly distracted by work and other private concerns.

I'm definitely going to look into GitHub sponsorships for the future, though.

**Reply by albertobeta** [reply]
@coldacid @arrigolupori my bad. I still have to share credentials with @coldacid and also we need to create a new donation link that Chris can access and use freely for PG-related activites.
Chris, I'll follow up via email to sort this out and move forward. Thanks

**Reply by arrigolupori** [reply]
@coldacid @albertobeta Do you guys need any help with refreshing the website in an open-source fashion? I have some time to dedicate to open-source projects and since I want to start a podcast soon, it makes a lot of sense for me to contribute!

I have 5+ years of experience designing websites and reading experiences on the web.
I'm a marketer turned web developer, so I'd be happy to help.

Anyway, let me know regarding the sponsorship :)
I'd love to see Podcast Generator keep rocking for many years to come!

**Reply by albertobeta** [reply]
It would be nice! let me discuss with @coldacid the next steps and we'll get back to you (probably next week)
